### ISSUE TITLE
Make highest heading level in document behave as H1

### ISSUE THREAD

**Issue by depascalis** [guopenghui/obsidian-quiet-outline#79]
I often have documents that doesn't use H2 and sometimes H3. If I have "Default expand level" set to "H1", then all levels in the outline will be collapsed which isn't desired in my case.

I suggest that the highest used heading level in the document would act as H1 in the outline pane. It's probably best that this is optional. Also, the H1-H5 options for "Default expand level" should probably be renamed to Level 1 - Level 5, or first, second, third.. to reduce the risk for confusion.

### Example
**Document Structure**
  - H3 - Heading A
    - H4 - Heading B
    - H4 - Heading C
  - H3 - Heading D
    - H4 - Heading E
      - H5 - Heading F
      - H5 - Heading G
---
**Outline Pane (Expand H1)** _Current behavior_ 
  - H3 - Heading A
  - H3 - Heading D
---
**Outline Pane (Expand H1)** _Suggested behavior_ 
  - H3 - Heading A
    - H4 - Heading B
    - H4 - Heading C
  - H3 - Heading D
    - H4 - Heading E

**Comment by skaviouz** [comment]
I also agree that the Default expand level should have their options "renamed".

Right now:
No expand --> H1
H1 --> H2
H2 --> H3
H3 --> H4
H4 --> H5
H5 --> H6

I feel regardless of this issues main point, you should rename H1-H5 to either H2-H6 or even:
- Collapsed headers
- Expand to children
- Expand to grandchildren
- Expand to great-grandchildren
- Expand to great-great-grandchildren
- Fully expanded headers

As OP said, right now the options should be renamed, especially considering that right now H1 means it expands until H2 and H5 means it expands until H6. It's confusing.
